build-support/bin/ci.py

```diff
@@ -420,37 +425,37 @@ def run_cargo_audit() -> None:
       die("Cargo audit failure")
 
 
-def run_unit_tests(*, oauth_token_path: Optional[str] = None) -> None:
+def run_unit_tests(*, oauth_token_path: Optional[str] = None, shard=None) -> None:
```

Comment:
Please add type hints for shard. I think Optional[str].

Reply:
Thanks! Adjusted.